src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -140,6 +140,8 @@ class NetworkController(settings: NetworkSettings,
       peer.handlerRef ! PeerConnectionHandler.CloseConnection
       peerManagerRef ! PeerManager.Disconnected(peer.socketAddress)
 
+    case Handshaked(remote, handshake) =>
```

Comment:
Do nothing here? Why do we send this message then?

Reply:
removed